base.mk

```diff
@@ -122,7 +122,7 @@ CFLAGS += -DITA_NO_ASSERT
 $(info Asserts will be switched off in debug mode due to code size limitation)
 endif
 else ifeq ($(BUILD), release)
-CFLAGS += -Os -fomit-frame-pointer
+CFLAGS += -Os -fomit-frame-pointer -flto
```

Comment:
If you're planning to add it anyway I think that it's ok to scrap my PR and you can work on it in your own pace. Just remember to fix missing implementation of qm_rtc_read().